xarray/tests/test_computation.py

```diff
@@ -998,6 +999,7 @@ def arrays_w_tuples():
         (arrays[1], arrays[1]),
         (arrays[2], arrays[2]),
         (arrays[2], arrays[3]),
+        (arrays[2], arrays[4]),
```

Comment:
```suggestion
        (arrays[2], arrays[4]),
        (arrays[4], arrays[2]),
```

Reply:
Is this to check that the ordering doesn't matter?

Reply:
Yes